packages/migrate/src/index.ts

```diff
@@ -1,22 +1,23 @@
-const fs = require("fs");
-const path = require("path");
-const glob = require("glob");
-const expect = require("@truffle/expect");
-const Config = require("@truffle/config");
-const Migration = require("./Migration");
-const emitEvent = require("./emitEvent");
-const inquirer = require("inquirer");
+import fs from "fs";
+import * as path from "path";
+import glob from "glob";
+import * as expect from "@truffle/expect";
+import Config from "@truffle/config";
+import { Migration } from "./Migration";
+import { emitEvent } from "./emitEvent";
+import inquirer from "inquirer";
+import type { Question } from "inquirer";
 
 /**
  *  This API is consumed by `@truffle/core` at the `migrate` and `test` commands via
  *  the `.runMigrations` method.
  */
-const Migrate = {
+export default {
   Migration: Migration,
   logger: null,
 
   promptToAcceptDryRun: async function (options) {
-    const prompt = [
+    const prompt: Question[] = [
```

Comment:
Should `options` above be typed? Ah, just noticed `promptToAcceptDryRun` is invoked without `options`.

Reply:
I think it probably should. It looks like it is a mistake not to include it because `emitEvent` uses it. Is this currently broken?

Reply:
I'm unsure if it is broken, though we should raise an issue and fix in follow-up work.